conf/documentation.conf

```diff
@@ -763,6 +763,8 @@ Describes type of the named interface.
  * "high-availability" is for an interface between two PacketFence servers dedicated to high-availability (drbd, corosync).
  * "portal" interface have the captive-portal running on them without the need of having any enforcement mechanism.
  * "radius" interface have the radius running (auth, acct, cli)
+ * "dhcpd" interface have the pfdhcp running.
```

Comment:
Like we don't call the radius interface radiusd, this should be called dhcp not dhcpd

Reply:
And it should be the "pfdhcp daemon"